packages/webpack-cli/package.json

```diff
@@ -27,8 +27,8 @@
     "@webpack-cli/info": "^1.0.1-alpha.4",
     "ansi-escapes": "^4.3.1",
     "chalk": "^3.0.0",
-    "command-line-args": "^5.1.1",
     "command-line-usage": "^6.1.0",
```

Comment:
I think we don't need that package too

Reply:
@evilebottnawi Good you pointed that out, we actually still rely on it for `webpack-cli help`. It still functions correctly, since we didn't change the `cli-flags` object format at all. I think it is fine actually to continue using it as it formats nicely and we can keep our current CLI flags object format is it is so that it continues to work with `command-line-usage`.

The alternative is to start using the built in commander help, we would just need to do some work making the output look nicer. Here is the difference:

Current webpack-cli help output using `command-line-usage`:
![webpack-cli-usage-help](https://user-images.githubusercontent.com/22580625/79824213-50423400-835b-11ea-967c-ddc7d356bd2f.png)

`commander` help:
![webpack-cli-commander-help](https://user-images.githubusercontent.com/22580625/79824224-59330580-835b-11ea-9084-577224d1d979.png)

Reply:
@Loonride Do we have API from commander for customize help output?

Reply:
> @Loonride Do we have API from commander for customize help output?

Yes indeed, https://github.com/tj/commander.js#custom-help